streamlit_folium/__init__.py

```diff
@@ -6,7 +6,6 @@
 import folium
 import folium.plugins
 import streamlit.components.v1 as components
-from folium.utilities import normalize
```

Comment:
I discovered that "normalize" just blindly deletes newlines, and this lead to a line being commented-out because of the comment on the line above it.